templates/zerver/help/view-an-image-at-full-size.md

```diff
@@ -10,6 +10,9 @@ To view the image in full size, click on the thumbnail preview of the image.
 Clicking it will open a lightbox modal with varying options depending on the
 media type.
 
+!!! tip ""
+    You can also view an image by pressing `v` on a selected message.
```

Comment:
Maybe "You can also view an image of the selected message by pressing `v`"

Reply:
I'm curious, why?

Reply:
I'm up to cut out "You can also" to make it shorter.

Reply:
When I read it right now, it makes it sound like you press `v` on the actual message. Like there's a literal `v` on the message which you press. "On a selected message" is describing the image, so it should be right after "image".

Reply:
There's nothing wrong with length. I was going for clarity.

Reply:
I get what you're saying, will keep it in mind as I go. Thanks!